Modules/Microsoft365DSC/DSCResources/MSFT_IntuneMobileThreatDefenseConnector/MSFT_IntuneMobileThreatDefenseConnector.psm1

```diff
@@ -0,0 +1,693 @@
+# https://learn.microsoft.com/en-us/graph/api/resources/intune-onboarding-mobilethreatdefenseconnector?view=graph-rest-1.0
+# https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.devicemanagement.administration/new-mgdevicemanagementmobilethreatdefenseconnector?view=graph-powershell-1.0
+
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        #region Intune parameters
+
+        [Parameter()]
+        [System.String]
+        $Id,
```

Comment:
We should make DisplayName mandatory since it needs to be the resource's key.

Reply:
DisplayName is not returned by the API and it's not getting exported either.